app/src/main/res/values/strings.xml

```diff
@@ -1,3 +1,4 @@
 <resources>
     <string name="app_name">Assos</string>
```

Comment:
the project is called assos, i don't know if the capital A is an issue

Reply:
The capital A is not an issue. You can see in a setting file that project name = "Assos"